src/core/createEventManager.js

```diff
@@ -32,23 +30,6 @@ export default ({
     }
   };
 
-  const addMetaTo = payload => {
-    const configOverrides = { orgId };
-
-    const dataCollection = Object.create(null);
-    assignIf(
-      dataCollection,
-      { synchronousValidation: true },
```

Comment:
Why are we removing synchronous validation? Is thing something that we plan on enabling through the experience cloud debugger?

Reply:
Yes, just sent out the invite for next week. But this wasn't working anyway, similar to how ORG ID override wasn't being applied.